docs/app-host/eventing.md

```diff
@@ -0,0 +1,87 @@
+---
+title: Eventing in .NET Aspire
+description: Learn how to use the .NET eventing features with .NET Aspire.
+ms.date: 10/30/2024
+---
+
+# Eventing in .NET Aspire
+
+In .NET Aspire, eventing allows you to publish and subscribe to events during various [app host](xref:dotnet/aspire/app-host#app-host-life-cycles) life cycles. Eventing is more flexible than life cycle events. Both let you run arbitrary code during event callbacks, but eventing offers finer control of event timing, publishing, and provides supports for custom events.
```

Comment:
Should this cross-reference with life cycle events, since it does a comparison to them.

Reply:
It links to them, that's the new App host - life cycles heading URL.

Reply:
Ah, yeah. The URL should enclose that life cycles text too. I'll fix that.